tests/conftest.py

```diff
@@ -4,4 +4,5 @@
 
 def pytest_collection_modifyitems(items):
     for item in items:
-        item.add_marker(pytest.mark.asyncio)
+        if isinstance(item, pytest.Function):
```

Comment:
Why is this needed?

Reply:
Because the mypy test cases (defined in the YAML file) break otherwise.